src/metadata.jl

```diff
@@ -52,3 +52,20 @@ function _image()
 
     return _CACHE[:IMAGE]
 end
+
+function Pkg.activate(jlso::JLSOFile, path=tempdir(); kwargs...)
+    ispath(path) || mkpath(path)
```

Comment:
The fact that we are doing this and not:
```suggestion
    mkdir(p"a"; recursive=true, exist_ok=true)
```
suggests something must be wrong in the FilePaths API.

Reply:
We don't depend on FilePathsBase here which I think is a reasonable choice for now.

Reply:
it does indeed not
```
shell> mkdir foo

shell> chmod 660 foo

shell> ls -lah
...
drw-rw----   2 oxinabox  staff    64B  5 Nov 18:24 foo

julia> mkpath("foo")
"foo"

shell> ls -lah
...
drw-rw----   2 oxinabox  staff    64B  5 Nov 18:24 foo

help?> mkpath
search: mkpath

  mkpath(path::AbstractString; mode::Unsigned = 0o777)

  Create all directories in the given path, with permissions mode. mode defaults to 0o777, modified by the current file creation mask. Return path.
```

I

Reply:
Looks like the answer that question is no https://github.com/JuliaLang/julia/blob/02deed30bfda45250ab0b6615b780e205e4811d9/base/file.jl#L221